### PR TITLE
Update English translation

### PR DESCRIPTION
**Pull request by jhult**
Grammar, phrasing, formatting. Feel free to edit/use/discard as you see fit.

### CODE REVIEW

File: i18n/en/czkawka_gui.ftl [hunk 1]
```diff
@@ -160,14 +160,14 @@ upper_notebook_excluded_directories = Excluded Directories
...
 
 upper_allowed_extensions_tooltip = 
-        Allowed extensions must be separated by commas(by default all are available).
+        Allowed extensions must be separated by commas (by default all are available).
 
-        Macros IMAGE, VIDEO, MUSIC, TEXT which adds multiple extensions at once are also available.
+        The following Macros, which add multiple extensions at once, are also available: IMAGE, VIDEO, MUSIC, TEXT.
 
-        Usage example  ".exe, IMAGE, VIDEO, .rar, 7z" - this means that image(e.g. jpg, png), video(e.g. avi, mp4), exe, rar and 7z files will be scanned.
+        Usage example  ".exe, IMAGE, VIDEO, .rar, 7z" - this means that images jpg, png), videos (e.g. avi, mp4), exe, rar, and 7z files will be scanned.
```
Comment (qarmin) [inline i18n/en/czkawka_gui.ftl]: Missing one ( around images

Comment (jhult) [reply i18n/en/czkawka_gui.ftl]: Fixed.

**Comment by jhult** [comment]
@qarmin, good catch. I fixed that and did a force push.

**Comment by qarmin** [comment]
Thanks!

**Pull request merged**